pkg/controller/containerjfr/resource_definitions/resource_definitions.go

```diff
@@ -63,7 +63,7 @@ func NewPodForCR(cr *rhjmcv1alpha1.ContainerJFR) *corev1.Pod {
 func NewCoreContainer(cr *rhjmcv1alpha1.ContainerJFR) corev1.Container {
 	return corev1.Container{
 		Name:  cr.Name,
-		Image: "quay.io/rh-jmc-team/container-jfr:0.4.6-debug",
+		Image: "quay.io/andrewazores/container-jfr:latest",
```

Comment:
Was this change intentional, or just for local testing?

Reply:
Local testing, fixed.